src/App.jsx

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable react/jsx-filename-extension */
+import React from 'react';
+import Person from './components/Person/Person';
+
+// eslint-disable-next-line
+const misha = {
+  name: 'Misha',
+  age: 37,
+  sex: 'm',
+  isMarried: true,
+  partnerName: 'Natasha',
+};
+
+// eslint-disable-next-line
+const olya = {
+  name: 'Olya',
+  sex: 'f',
+  isMarried: true,
+  partnerName: 'Maksym',
+};
+
+// eslint-disable-next-line
+const alex = {
+  name: 'Alex',
+  age: 25,
+  sex: 'm',
+  isMarried: false,
+};
+
+const App = () => (
+  <div className="App">
+    <Person {... misha} />
+    <Person {... olya} />
+    <Person {... alex} />
```

Comment:
Please, read this:
http://mateacademy.slack.com/archives/C03C2C13KRA/p1657005525754959

Reply:
thanks! 😊